app/javascript/Pages/Home.jsx

```diff
@@ -1,37 +1,272 @@
 import React, { Component } from "react";
 import Navbar from "./components/Navbar/Navbar";
 import Footer from "./components/Footer/Footer";
-import { Head, Link } from "@inertiajs/inertia-react";
-import "../stylesheets/Home.css";
-
-class Homepage extends Component {
-  render() {
-    return (
-      <main>
-        <Navbar current_user={this.props.current_user} />
-
-        <img
-          src={require("../images/banner.svg")}
-          alt=""
-          role="presentation"
-          className="hero-img"
-        />
-        <Head title="Welcome" />
-        <p>Hello, welcome to your first Inertia app!</p>
-        <p className="paragraph">This is cool!</p>
-
-        <Link
-          href="/logout"
-          method="delete"
-          headers={window.defaultHeaders}
-          as="button"
+import { Head } from "@inertiajs/inertia-react";
+import { useState, useEffect } from "react";
+import SocialCard from "./components/SocialCard";
+import "../stylesheets/home.scss";
+import ButtonDifficultyFilter from "./components/ButtonDifficultyFilter";
+import CardCategory from "./components/CardCategory";
+import CardPopular from "./components/CardPopular";
+import axios from "axios";
+import ButtonFilter from "./components/ButtonFilter";
+import PropTypes from "prop-types";
+import { useTranslation } from "react-i18next";
```

Comment:
**issue (non-blocking)**: The order of file imports should not be random. Usually we do 1. "libraries", 2. "components from libraries", 3. "local components".
Also, related, does this project have ESLint configured? Some configurations also help with this by showing the order you should follow. (https://airbnb.io/javascript/react/ is a good base)